cmd/ipfs-cluster-ctl/main.go

```diff
@@ -508,12 +505,20 @@ peers should pin this content.
 							rplMax = rpl
 						}
 
-						ci, cerr := globalClient.PinPath(path.String(), rplMin, rplMax, c.String("name"))
+						opts := api.PinOptionsWithPath{
+							Path: path.String(),
+							PinOptions: api.PinOptions{
+								ReplicationFactorMin: rplMin,
+								ReplicationFactorMax: rplMax,
+								Name:                 c.String("Name"),
+							},
+						}
+						pin, cerr := globalClient.PinPath(opts)
 						if cerr != nil {
 							formatResponse(c, nil, cerr)
 							return nil
 						}
-
+						ci := pin.Cid
```

Comment:
should adjust `handlePinResponseFormatFlags()`.  If `--no-status` is passed, you should print the returned pin object.

Reply:
Cool, now it looks like 
```
$ ipfs-cluster-ctl pin add /ipfs/QmbkfHeLFHzKR9723HVgktuFb8dURv9jG2ez6SLq1gjQfG
QmbkfHeLFHzKR9723HVgktuFb8dURv9jG2ez6SLq1gjQfG :
    > kishansagathiya : PINNING | 2019-02-05T02:37:27Z
    > kishansagathiya : PINNING | 2019-02-05T02:37:24Z
$ ipfs-cluster-ctl pin add /ipfs/QmbkfHeLFHzKR9723HVgktuFb8dURv9jG2ez6SLq1gjQfG --no-status
QmbkfHeLFHzKR9723HVgktuFb8dURv9jG2ez6SLq1gjQfG |  | PIN | Repl. Factor: -1 | Allocations: [everywhere] | Recursive
$ 
```